src/components/organisms/GlobalNav.vue

```diff
@@ -50,7 +55,7 @@ export default class GlobalNav extends Vue {
     height: 20px;
     fill: #999;
 
-    &._isActive {
+    a._isActive & {
```

Comment:
```suggestion
    > a._isActive & {
```

要素セレクタを使う場合は可能なら影響範囲を狭めておこう。